packages/workspace/src/browser/workspace-service.ts

```diff
@@ -115,9 +115,9 @@ export class WorkspaceService implements FrontendApplicationContribution {
         return this._workspace;
     }
 
-    protected readonly onWorkspaceChangeEmitter = new Emitter<FileStat[]>();
-    get onWorkspaceChanged(): Event<FileStat[]> {
-        return this.onWorkspaceChangeEmitter.event;
+    protected readonly onWorkspaceRootsChangeEmitter = new Emitter<FileStat[]>();
+    get onWorkspaceRootsChanged(): Event<FileStat[]> {
```

Comment:
please revert changes to APIs, `WorkspaceService` is used quite a lot

Reply:
OK, I've renamed it back to onWorkspaceChanged and force-pushed.

As a consumer of Theia's API in our product, I would rather names are maintained to be clear and avoid ambiguity even if this means having to do the odd rename when we upgrade, so I made the decision to rename.   However I realize there are other users and concerns so I am happy to revert the name.